interface/forms/requisition/new.php

```diff
@@ -0,0 +1,368 @@
+<?php
+/**
+ *  Lab Requisition Form
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
+
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/options.inc.php");
+require_once("$srcdir/htmlspecialchars.inc.php");
```

Comment:
above line can be removed (this is already always included).

Reply:
@juggernautsei ,
There was a miscommunication by me here. Only the line bringing in htmlspecialchars.inc.php can be removed. All the other require_once lines need to stay :)